introduction.md

```diff
@@ -3,4 +3,9 @@ Hi! I'm Christine. I like programming because of the many opportunities it offer
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?